locomotion/trajectory.py

```diff
@@ -43,13 +43,15 @@ def getDerivatives(X, axis = 0):
   return dX
 
 
-def smooth(X):
+def smooth(X, frame_rate = 24):
```

Comment:
Let's not set a default value for frame_rate since we don't have a default camera.

Reply:
Alright, I'll make the change and make another Pull Request